scripts/cluster-up-minikube.sh

```diff
@@ -28,7 +28,7 @@ touch ~/.kube/config
 export KUBECONFIG=$HOME/.kube/config
 export PATH=${PATH}:${GOPATH:?}/bin
 
-MINIKUBE_VERSION=v0.28.0
+MINIKUBE_VERSION=v0.28.1
 KUBERNETES_VERSION=v1.10.0
```

Comment:
Curious why k8s version is 1.10.0 I thought we moved to 1.11, am I wrong?

Reply:
Minikube only supports up to v1.10.0.